README.md

```diff
@@ -141,6 +141,12 @@ You can demo this feature in our [PaymentSheet example app](Example/PaymentSheet
 
 We welcome contributions of any kind including new features, bug fixes, and documentation improvements. Please first open an issue describing what you want to build if it is a major change so that we can discuss how to move forward. Otherwise, go ahead and open a pull request for minor changes such as typo fixes and one liners.
 
+### Generating workspace
+
+We use Tuist to generate a workspace. Please install Tuise following the [Get started guide](https://docs.tuist.io/tutorial/get-started)
+
+After installation run `tuist generate` to generate the Stripe workspace.
+
```

Comment:
These instructions are already included in the Building from source section, so they are redundant, please remove.

Reply:
Sorry must have missed this